org.eclipse.winery.frontends/app/tosca-management/src/app/wineryUtils/utils.ts

```diff
@@ -21,6 +21,30 @@ export class Utils {
         return Object.keys(object).length === 0;
     }
 
+    public static isLoading(map: {[key: string]: boolean}): boolean {
+        return Object.keys(map).some(k => map[k]);
+    }
+
+    public static doubleEncodeNamespace(namespace: string): string {
+        return encodeURIComponent(encodeURIComponent(namespace));
+    }
+
+    public static getFrontendPath(toscaType: string, namespace: string, localname: string) {
+        // TODO: this should be double encode?!
```

Comment:
Normally it should, however, it may be the case that Angular is automatically encoding the already encoded elements a second time, thus it works in this case if its only encoded once. 
Does it work currently?